apptools/undo/tests/test_command_stack.py

```diff
@@ -0,0 +1,119 @@
+# -----------------------------------------------------------------------------
+#
+#  Copyright (c) 2015, Enthought, Inc.
+#  All rights reserved.
+#
+#  This software is provided without warranty under the terms of the BSD
+#  license included in enthought/LICENSE.txt and may be redistributed only
+#  under the conditions described in the aforementioned license.  The license
+#  is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+#  Thanks for using Enthought open source!
+#
+# -----------------------------------------------------------------------------
+
+from traits.testing.unittest_tools import unittest
+
+from apptools.undo.api import CommandStack, UndoManager
+from testing_commands import SimpleCommand, UnnamedCommand
```

Comment:
Any particular reason for putting these in a separate module? It makes the tests harder to follow.